packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -546,6 +546,23 @@ const api = new apigateway.RestApi(this, 'books', {
 })
 ```
 
+You can also use the `deployOptions` property to configure
```

Comment:
Wondering if this would be better placed in the section 'Default Integration and Method Options', given that is a 'method option'. What do you think?

Reply:
I considered that at first. The reason I have it here is because both of these examples are configuring options at the "stage" level, not the method level.

If you were to configure method throttling along with the examples in 'Default Integration and Method Options', that value would apply to that method in all stages. For me, personally, that was easy to figure out. Discovering how to set the defaults for the stage/deployment was what was difficult.